Agreements/Internal/MAW_MAE.txt

```diff
@@ -17,4 +17,6 @@ COPX:*:*:CREWE:EGBB:*:Scottish PC West:Scottish PC East:*:20000:CREWE25
 COPX:*:*:NUGRA:EGNX:*:Scottish PC West:Scottish PC East:*:20000:NUGRA15
 
 COPX:*:*:NOKIN:EGBB:*:Scottish PC West:Scottish PC East:*:20000:NOKIN
-COPX:*:*:NOKIN:EGNX:*:Scottish PC West:Scottish PC East:*:20000:NOKIN
\ No newline at end of file
+COPX:*:*:NOKIN:EGNX:*:Scottish PC West:Scottish PC East:*:20000:NOKIN
+
+COPX:*:*:POL:EGNT:*:Scottish PC West:Scottish PC East:*:26000:POL
```

Comment:
This line is already here (line 5) - just change the level

Reply:
Fixed